refactoring.api/src/org/netbeans/modules/refactoring/api/RefactoringSession.java

```diff
@@ -82,29 +80,23 @@ public static RefactoringSession create(@NonNull String description) {
         return new RefactoringSession(description);
     }
 
-
     /**
-     * process all elements from elements bags,
-     * do all fileChanges
-     * and call all commits
+     * process all elements from elements bags, do all fileChanges and call all
+     * commits
+     *
      * @param saveAfterDone save all if true
      * @return instance of Problem or null, if everything is OK
      */
     @CheckForNull
     public Problem doRefactoring(final boolean saveAfterDone) {
-        return Utilities.runWithOnSaveTasksDisabled(new Action<Problem>() {
-            @Override public Problem run() {
-                return reallyDoRefactoring(saveAfterDone);
-            }
-        });
+        return Utilities.runWithOnSaveTasksDisabled(() -> reallyDoRefactoring(saveAfterDone));
     }
-    
+
     private Problem reallyDoRefactoring(boolean saveAfterDone) {
-        long time = System.currentTimeMillis();
-        
+        final long start = System.currentTimeMillis();
         Iterator it = internalList.iterator();
```

Comment:
The type of `it` could be `Iterator<RefactoringElementImplementation>`.

Reply:
It's ok as it is.